dao/src/main/scala/za/co/absa/enceladus/menasplugin/MenasCredentials.scala

```diff
@@ -18,13 +18,36 @@ package za.co.absa.enceladus.menasplugin
 import java.io.File
 
 import com.typesafe.config.ConfigFactory
+import org.apache.hadoop.fs.FileSystem
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.Path
+import org.apache.spark.sql.SparkSession
+import org.slf4j.LoggerFactory
+import za.co.absa.enceladus.utils.fs.FileSystemVersionUtils
 
 object MenasCredentials {
 
-  def fromFile(path: String): MenasCredentials = {
-    val conf = ConfigFactory.parseFile(new File(replaceHome(path)))
+  private val logger = LoggerFactory.getLogger(this.getClass)
+  
+  /**
+   * Retrieves Menas Credentials from file either on local file system or on HDFS
+   */
+  def fromFile(path: String)(implicit spark: SparkSession): MenasCredentials = {
+    val conf = if(path.startsWith("hdfs://")) {
```

Comment:
This is not an issue, just a thought. In Cobrix we used a slightly different convention. If a path begins with `file://` the file will be looked up locally, otherwise it will be looked up in HDFS.

The reason for this is that when you pass a path to Spark, it will be handled through Hadoop readers, even in local mode. So in Cobrix we handle copybook paths Spark's way by default. The `file://` prefix is something you do not pass to Spark since it does not work with local files. Furthermore `hdfs://` requires a node name or a cluster name, isn't it? If we omit the prefix this information will be retrieved from Hadoop config.

Maybe both conventions are not perfect and `hdfs://` tells to look exclusively in HDFS, `file://` to look exclusively local, but if the prefix is not specified, maybe check both places... not sure.

Reply:
It is true that hdfs sometimes requires // and a URL, sometimes ///. It can be more error prone

Reply:
Yeah I guess it would be definitely better to default to HDFS.

The reason I went for a local FS first is that I didn't wanna make a breaking change (given that this was used previously). 

Ideas?

Reply:
Cool, so let's default to HDFS. But I'll change that in the next PR for #444